CMakeLists.txt

```diff
@@ -273,14 +273,19 @@ endif()
 
 add_definitions("-DBLOCKCHAIN_DB=${BLOCKCHAIN_DB}")
 
-find_package(Libunwind)
 # Can't install hook in static build on OSX, because OSX linker does not support --wrap
 # On ARM, having libunwind package (with .so's only) installed breaks static link.
-if(LIBUNWIND_FOUND AND NOT (STATIC AND (APPLE OR ARM)))
-  set(DEFAULT_STACK_TRACE ON)
-else()
+if(STATIC OR APPLE OR ARM)
```

Comment:
I may need to change this if we want it to build with libunwind on ARM/OSX in non-static mode